### DISCUSSION TITLE
backtick math failures

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#39238] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Math on GitHub continues to have many problems. I just found another one. The tripe-backtick math (which I thought was safe to use) produces the following error:

````markdown
```math
a < b > c
```
````

```math
a < b > c
```

Screenshot:

![screenshot](https://user-images.githubusercontent.com/181628/202145468-0bc96079-2b9c-47dc-838b-8641a71f14af.png)

**Comment by Preetiraj3697** [comment]
There are two points I missed. First, as [this answer](https://stackoverflow.com/a/46765337/7850499) pointed out, I should add the following code to _includes/*.html


<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        skipTags: ['script', 'noscript', 'style', 'textarea', 'pre'],
        inlineMath: [['\\(','\\)'], ['$', '$']]
      }
    });
  </script>
  <script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>

Second, I should replace all inline math surrounded by $...$ with \\(...\\), as well as all math blocks surrounded by $$...$$ with \\[...\\].

However, I still cannot figure out why everything works fine on my local computer before.

**Reply by nschloe** [reply]
This answer doesn't seem to refer to the question. Misclick?

**Comment by nschloe** [comment, marked as answer]
This appears to work now (Nov 30, 2022).
